src/scripts/main.js

```
@@ -1,360 +1,52 @@
-    died: 1781,
-    fatherName: 'Pieter de Decker',
-    motherName: 'Livina Haverbeke',
-    slug: 'petronella-de-decker-1731',
-  },
-  {
-    name: 'Livina Sierens',
-    sex: 'f',
-    born: 1761,
-    died: 1826,
-    fatherName: 'Jan Sierens',
-    motherName: 'Maria van Waes',
-    slug: 'livina-sierens-1761',
-  },
-  {
-    name: 'Laurentia Haverbeke',
-    sex: 'f',
-    born: 1710,
-    died: 1786,
-    fatherName: 'Jan Haverbeke',
-    motherName: 'Maria de Rycke',
-    slug: 'laurentia-haverbeke-1710',
-  },
-  {
-    name: 'Carel Haverbeke',
-    sex: 'm',
-    born: 1796,
-    died: 1837,
-    fatherName: 'Pieter Antone Haverbeke',
-    motherName: 'Livina Sierens',
-    slug: 'carel-haverbeke-1796',
-  },
-  {
-    name: 'Elisabeth Hercke',
-    sex: 'f',
-    born: 1632,
-    died: 1674,
-    fatherName: 'Willem Hercke',
-    motherName: 'Margriet de Brabander',
-    slug: 'elisabeth-hercke-1632',
-  },
-  {
-    name: 'Jan Haverbeke',
-    sex: 'm',
-    born: 1671,
-    died: 1731,
-    fatherName: 'Lieven Haverbeke',
-    motherName: 'Elisabeth Hercke',
-    slug: 'jan-haverbeke-1671',
-  },
-  {
-    name: 'Anna van Hecke',
-    sex: 'f',
-    born: 1607,
-    died: 1670,
-    fatherName: 'Paschasius van Hecke',
-    motherName: 'Martijntken Beelaert',
-    slug: 'anna-van-hecke-1607',
-  },
-  {
-    name: 'Maria Sturm',
-    sex: 'f',
-    born: 1835,
-    died: 1917,
-    fatherName: 'Charles Sturm',
-    motherName: 'Seraphina Spelier',
-    slug: 'maria-sturm-1835',
-  },
-  {
-    name: 'Jacobus Bernardus van Brussel',
-    sex: 'm',
-    born: 1736,
-    died: 1809,
-    fatherName: 'Jan van Brussel',
-    motherName: 'Elisabeth Haverbeke',
-    slug: 'jacobus-bernardus-van-brussel-1736',
-  },
-];
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+  if (!table) {
+    return;
+  }
```

Consider improving the missing-table guard: currently the code silently returns when the table is not found. To make debugging easier, log an informative error message before returning (optional): `if (!table) { console.error('Dashboard table not found'); return; }`.